scripts/lib/CIME/SystemTests/ncr.py

```diff
@@ -1,111 +1,68 @@
-            self.clean_build()
-            self.build_indv(sharedlib_only, model_only)
-            shutil.move("{}/{}.exe".format(exeroot,cime_model),
-                        "{}/{}.exe.NCR{}".format(exeroot,cime_model,bld))
-            lock_file("env_build.xml", newname="env_build.NCR{}.xml".format(bld))
-            lock_file("env_mach_pes.xml", newname=machpes)
-
-        # Because mira/cetus interprets its run script differently than
-        # other systems we need to copy the original env_mach_pes.xml back
-        restore(machpes1, newname="env_mach_pes.xml")
-
-    def run_phase(self):
-        os.chdir(self._caseroot)
-
-        exeroot = self._case.get_value("EXEROOT")
-        cime_model = CIME.utils.get_model()
-
-        # Reset beginning test settings
-        expect(is_locked("env_mach_pes.NCR1.xml"),
-               "ERROR: LockedFiles/env_mach_pes.NCR1.xml does not exist\n"
-               "   this would been produced in the build - must run case.test_build")
-
-        restore("env_mach_pes.NCR1.xml", newname="env_mach_pes.xml")
-        restore("env_build.NCR1.xml", newname="env_build.xml")
-        shutil.copy("{}/{}.exe.NCR1".format(exeroot, cime_model),
-                    "{}/{}.exe".format(exeroot, cime_model))
-
-
-        stop_n      = self._case.get_value("STOP_N")
-        stop_option = self._case.get_value("STOP_OPTION")
-
-        self._case.set_value("HIST_N", stop_n)
-        self._case.set_value("HIST_OPTION", stop_option)
-        self._case.set_value("CONTINUE_RUN", False)
-        self._case.set_value("REST_OPTION", "none")
-        self._case.flush()
-
-        #======================================================================
-        # do an initial run test with NINST 1
-        #======================================================================
-        logger.info("default: doing a {} {} with NINST1".format(stop_n, stop_option))
-        self.run_indv()
-
-        #======================================================================
-        # do an initial run test with NINST 2
-        # want to run on same pe counts per instance and same cpl pe count
-        #======================================================================
-
-        os.remove("{}/{}.exe".format(exeroot, cime_model))
-        shutil.copy("{}/{}.exe.NCR2".format(exeroot, cime_model),
-                    "{}/{}.exe".format(exeroot, cime_model))
-        restore("env_build.NCR2.xml", newname="env_build.xml")
-
-        logger.info("default: doing a {} {} with NINST2".format(stop_n, stop_option))
-        self.run_indv(suffix="multiinst")
-
-        # Compare
-        self._component_compare_test("base", "multiinst")
+        SystemTestsCompareTwo.__init__(self, case,
+                                       separate_builds = True,
+                                       run_two_suffix = "singleinst",
+                                       run_one_description = "default build",
```

Comment:
I have them reversed because I'm uncertain it'll work otherwise; but I'll try it and see if it works